pkgs/by-name/cx/cxx-rs/package.nix

```diff
@@ -2,13 +2,13 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "cxx-rs";
-  version = "1.0.94";
+  version = "1.0.131";
 
   src = fetchFromGitHub {
     owner = "dtolnay";
     repo = "cxx";
     rev = version;
-    sha256 = "sha256-h6TmQyxhoOhaAWBZr9rRPCf0BE2QMBIYm5uTVKD2paE=";
+    sha256 = "sha256-KQlbJvULdc94SM0sx6JtukZPpaX4Gojc6Qgr20V3/VI=";
```

Comment:
```suggestion
    hash = "sha256-KQlbJvULdc94SM0sx6JtukZPpaX4Gojc6Qgr20V3/VI=";
```

Reply:
I'm sorry, but I won't wait ~20 instead of ~6 hours more just for this small fix. I don't even maintain or care too much for this package, I just ran into this need of updating by chance.